java/src/main/java/com/leecampbell/cod/application/SparkApplication.java

```diff
@@ -0,0 +1,76 @@
+package com.leecampbell.cod.application;
+
+import static spark.Spark.*;
+
+import java.math.BigDecimal;
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.UUID;
+import com.google.gson.Gson;
+import com.leecampbell.cod.domain.contracts.*;
+import com.leecampbell.cod.domain.model.Loan;
+import com.leecampbell.cod.domain.services.CreateLoanCommandHandler;
+import com.leecampbell.cod.domain.services.Repository;
+
+import spark.Request;
+import spark.Response;
+
+public class SparkApplication {
+    private static final Gson gson = new Gson();
+
+    public static void main(String[] args) {
+        get("/hello", (req, res) -> "Hello, World! GFY");
+
+        post("/Loan", "application/json", SparkApplication::createLoan);
+        get("/foo", "application/json", (req, res) -> {
+            Repository repository = new FakeRepository();
+            CreateLoanCommandHandler handler = new CreateLoanCommandHandler(repository);
+            CreateLoanCommand command = CreateCommand();
+
+            Receipt r = handler.handle(command);
+            return r.aggregateId().toString();
+        });
+    }
+
+    private static Object createLoan(Request request, Response response) {
+        String requestBody = request.body();
+        CreateLoanModel requestModel = gson.fromJson(requestBody, CreateLoanModel.class);
+        CreateLoanCommand command = requestModel.toCommand();
+
+        Repository repository = new FakeRepository();
+        CreateLoanCommandHandler handler = new CreateLoanCommandHandler(repository);
+        Receipt receipt = handler.handle(command);
+
+        LoanCreatedModel responseModel = new LoanCreatedModel();
+        responseModel.loanId = receipt.aggregateId();
+        return gson.toJson(responseModel);
```

Comment:
can use json response transformers here if you want
http://sparkjava.com/documentation#response-transformer